code/modules/research/mechanic/flatpack.dm

```diff
@@ -147,6 +147,13 @@
 		if((stacked.len + stacking.stacked.len + 2) >= MAX_FLATPACK_STACKS) //how many flatpacks we can in a stack (including the bases)
 			to_chat(user, "You can't stack flatpacks that high.")
 			return
+		if(user.incapacitated() || user.lying)
```

Comment:
```
/mob/proc/isStunned() //Because we have around four slighly different stunned variables for some reason.
	if(isUnconscious() || paralysis || stunned || knockdown)
		return 1

/mob/proc/incapacitated()
	if(isStunned() || restrained())
		return 1
```

I think the lying and knockdown checks should be removed from literally everything that have them, but that's outside the scope of this PR.